test/java/org/chromium/net/CertificateVerificationTest.java

```diff
@@ -33,40 +36,89 @@ public void tearDown() throws Exception {
   }
 
   @Test
-  public void testVerifyCertificateCall() throws Exception {
-    String[] fakeCertChain1 = new String[] {"fake cert"};
-    byte[][] certChain1 = new byte[][] {fakeCertChain1[0].getBytes()};
-
-    String[] fakeCertChain2 = new String[] {"fake cert", "another fake cert"};
-    byte[][] certChain2 = new byte[][] {fakeCertChain2[0].getBytes(), fakeCertChain2[1].getBytes()};
+  public void testChainWithNonRootCertificate() throws Exception {
+    final String[] fakeCertChain = new String[] {"fake cert"};
+    final byte[][] certChain = new byte[][] {fakeCertChain[0].getBytes()};
+    final String authType = "";
+    final String host = "";
```

Comment:
Your call, but I would be inclined to turn these 4 variables into members of the test class to avoid repeating them each time. But totally optional.